automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -20,6 +20,8 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   def editMethodConfig(newName: Option[String] = None, newSnapshotId: Option[Int] = None, newRootEntityType: Option[String] = None,
                        inputs: Option[Map[String, String]] = None, outputs: Option[Map[String, String]] = None) = {
     ui.openEditMode()
+    await spinner "Loading attributes..."
```

Comment:
This seems brittle. What if we decide to capitalize that "a"?

Reply:
Unfortunately WebDriver only gives us XPath 1.0 so `matches()` and `lower-case()` are out. There's an ugly way to do a case-insensitive `conatins()` using `translate()` which I guess is better than nothing. I'll add that and we'll see over time if it's otherwise too brittle in practice.

Reply:
I just meant it as an example. I don't like the idea of asserting that fairly arbitrary text is a certain way.

Reply:
Okay. I agree it's not great. I'll think about how to make this better and am open to suggestions.